components/lib/chip/Chip.spec.js

```diff
@@ -2,40 +2,42 @@ import '@testing-library/jest-dom';
 import { fireEvent, render } from '@testing-library/react';
 import { Chip } from './Chip';
 
-const snapshot = (props, name) => expect(render(<Chip {...props} />).container).toMatchSnapshot(name);
+import { snapshot } from '../../test';
 
 describe('Chip', () => {
-    test('check snapshots', () => {
-        snapshot({}, 'default');
-        snapshot({ image: 'http://google.com/my.png' }, 'image');
-        snapshot({ image: 'http://google.com/my.png', imageAlt: 'jest' }, 'image and alt');
-        snapshot({ icon: 'pi pi-check' }, 'icon');
-        snapshot({ label: 'jest' }, 'label');
-    });
+    snapshot(<Chip />, 'default');
+    snapshot(<Chip image="http://google.com/my.png" />, 'image');
+    snapshot(<Chip image="http://google.com/my.png" imageAlt="jest" />, 'image and alt');
+    snapshot(<Chip icon="pi pi-check" />, 'icon');
+    snapshot(<Chip label="jest" />, 'label');
     test('when removable is true it returns with remove icon', () => {
         // Arrange
         const removeOn = jest.fn();
         const { container } = render(<Chip removable onRemove={removeOn} />);
+
+        expect(container).toMatchSnapshot('before remove');
```

Comment:
Here is example how to pass a name to the `toMatchSnapshot` call, so that two snapshots are made in the same test. Here is makes sense to snapshot before and after the action, as the component rendered differently.